### PR TITLE
Feature/66712

### PR DESCRIPTION
**Pull request by JKrupinski**
[#66712](https://dev.snowdog.pro/issues/66712)

### CODE REVIEW

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+                        class="
+                            list__item
+                            item<?php if ($_typeCode == $_ccType) : ?>
+                            _active<?php endif; ?>
+                        "
+                        data-type="<?= $block->escapeHtmlAttr($_typeCode); ?>"
+                    >
+                        <div class="lazyload-wrapper">
+                            <img
+                                src="<?= $block->getViewFileUrl('Magento_Payment::images/cc/' . strtolower($_typeCode) . '.png'); ?>"
```
Comment (anqaka) [inline ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:324]: if lazyload is used, shouldn't be `data-src` attribute used?

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+                    >
+                        <?= $block->escapeHtml(__('Street Address %1', $lineNo)) ?>
+                    </label>
+                    <input
+                        type="text"
+                        name="billing[street][]"
+                        value="<?= $block->escapeHtmlAttr($block->getStreetLine($lineNo)) ?>"
+                        title="<?= $block->escapeHtmlAttr(__('Street Address %1', $lineNo)) ?>"
+                        id="street_<?= $lineNo ?>"
+                        class="input__field <?= $_streetValidationClass ?>"
```
Comment (anqaka) [inline ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:154]: split classes please

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml [hunk 3]
```diff
@@ -0,0 +1,615 @@
...
+
+        <div
+            class="
+                input
+                type
+                required
+            "
+        >
+            <?php $_ccType = $block->getCard()->getAdditional('cc_type') ?>
+            <ul class="credit-card-types list list--horizontal">
```
Comment (anqaka) [inline ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:312]: split classes please

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml [hunk 4]
```diff
@@ -0,0 +1,615 @@
...
+                            item<?php if ($_typeCode == $_ccType) : ?>
+                            _active<?php endif; ?>
+                        "
+                        data-type="<?= $block->escapeHtmlAttr($_typeCode); ?>"
+                    >
+                        <div class="lazyload-wrapper">
+                            <img
+                                src="<?= $block->getViewFileUrl('Magento_Payment::images/cc/' . strtolower($_typeCode) . '.png'); ?>"
+                                alt="<?= $block->escapeHtmlAttr($_typeName); ?>"
+                                class="image lazyload width-auto"
```
Comment (anqaka) [inline ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:326]: split classes please

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml [hunk 5]
```diff
@@ -0,0 +1,615 @@
...
+                                class="
+                                    modal__content
+                                    padding-hz-xl
+                                    padding-vc-xl
+                                "
+                            >
+                                <img
+                                    src="<?= $block->escapeUrl($block->getViewFileUrl('Magento_Checkout::cvv.png')) ?>"
+                                    alt="<?= $block->escapeHtmlAttr(__('Card Verification Number Visual Reference')) ?>"
+                                    class="image width-auto"
```
Comment (anqaka) [inline ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:519]: is it worth to use lazyload here?

File: ParadoxLabs_TokenBase/templates/customer/cards.phtml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                        <?php if (!empty($card->getType())): ?>
+                            <img
+                                class="
+                                    credit-card-type
+                                    payment-icon
+                                    image
+                                    width-auto
+                                "
+                                src="<?= $block->getViewFileUrl('Magento_Payment::images/cc/' . strtolower($card->getType()) . '.png'); ?>"
+                                alt="<?= $block->escapeHtmlAttr($block->getCcTypeLabel($card)); ?>"
```
Comment (anqaka) [inline ParadoxLabs_TokenBase/templates/customer/cards.phtml:45]: is it worth to use here lazyload?

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml (hunk 1 above)
Comment (JKrupinski) [reply ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:324]: You've right but we don't need lazy loading here, I'll remove it

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml (hunk 4 above)
Comment (JKrupinski) [reply ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:326]: I'll remove `lazyload` class

File: ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml (hunk 5 above)
Comment (JKrupinski) [reply ParadoxLabs_Authnetcim/templates/customer/form/cc.phtml:519]: Nope

File: ParadoxLabs_TokenBase/templates/customer/cards.phtml (hunk 6 above)
Comment (JKrupinski) [reply ParadoxLabs_TokenBase/templates/customer/cards.phtml:45]: It's not lazyloaded

Comment (JKrupinski) [reply ParadoxLabs_TokenBase/templates/customer/cards.phtml:45]: Nope

**Comment by JKrupinski** [comment]
I've moved one template from BBQ theme (ParadoxLabs_Authnetcim/web/template/payment/cc.html)

**Pull request merged**